config/jobs/build-drivers/build-drivers.yaml

```diff
@@ -195,27 +197,29 @@ periodics:
         env:
         - name: AWS_REGION
           value: eu-west-1
-        image: 292999226676.dkr.ecr.eu-west-1.amazonaws.com/test-infra/build-drivers:latest
+        image: 292999226676.dkr.ecr.eu-west-1.amazonaws.com/test-infra/build-drivers:s3-publish
         imagePullPolicy: Always
         securityContext:
           privileged: true
   - name: build-drivers-debian-periodic
     decorate: true
-    cron: "0 03 * * 4"
+    interval: 1h
+    # cron: "0 03 * * 4"
```

Comment:
Ok, this periodic `ProwJob` seems to have worked 👉  https://download.falco.org/?prefix=driver/5c0b863ddade7a45568c0ac97d037422c9efb750/

Anyways, I was expecting the presubmit build jobs to run on this PR...

Reply:
For testing I used the shortest running job, and set the interval to 1H so I could figure out any bugs. I just changed it back so the upload only runs on presubmits in my new commit #43931df